src/main/java/seedu/address/logic/commands/ConfirmedClearCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.model.RecipeBook;
+
+/**
+ * Clears the recipe book after user confirms or the user specified a forced clear.
+ */
+public class ConfirmedClearCommand extends ClearCommand {
+    public static final String COMMAND_WORD = "yes";
+    public static final String COMMAND_NAME = "Confirmed Clear";
+    public static final String MESSAGE_SUCCESS = "Recipe book has been cleared!";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command! Confirm clear by typing in yes or "
```

Comment:
LGTM 🌮  Just wna ask, if a user inputs an unknown command, the prompt to get confirmation for clearance will be repeated?

Reply:
Nope, it'll throw an exception and exit the confirmation state. Can refer to `RecipeBookParser` file yas